tilauskasittely/kopioi_tilaus.inc

```diff
@@ -68,6 +76,20 @@ if (!function_exists("kopioi_tilaus")) {
       if ($field == "kate_korjattu") {
         $values .= ", NULL";
       }
+      elseif ($field == "erpcm" and $meapurow["erapvmkasin"] != '') {
+        $pp = (int) substr($monistarow[$i], 8, 2);
+        $kk = (int) substr($monistarow[$i], 5, 2);
+        $vv = (int) substr($monistarow[$i], 0, 4);
+
+        // Jos on sy�tetty er�p�iv�m��r�st� vain p�iv�, taiotaan siit� kokonainen p�iv�m��r� seuraavalle kuukaudelle
+        if ($pp > 0 and $pp == 0 and $vv == 0) {
+          $seuraavankuunpvm = date("Y-m-{$pp}", strtotime('+1 month'));
```

Comment:
@tonyselkamo @mikamyllynen Pitäsikö alegortmin kuitenkin olla sellainen, että mennään kuukaudella etteenpäin vain jos $pp <= date("d")? Muuten eräpäivä ois suoraan date("Y-m-{$pp}").